_episodes/01-relational-database.md

```diff
@@ -30,7 +30,7 @@ keypoints:
 
 A relational database is a collection of data items organised as a set of tables. Relationships can be defined between the data in one table and the data in another or many other tables. The relational database system will provide mechanisms by which you can query the data in the tables, re-assemble the data in various ways without altering the data in the actual tables. 
 This querying is usually done using SQL (Structured Query Language). SQL allows a great many queries to be constructed from the use of only a few keywords.
-You could have a relational database with only one table, but then you would’t have any relationships and it would be more like a spreadsheet. 
+You could have a relational database with only one table, but then you wouldn’t have any relationships and it would be more like a spreadsheet. 
```

Comment:
```suggestion
You could have a relational database with only one table, but then you would not have any relationships and it would be more like a spreadsheet. 
```

expanding the negative contraction to make the content more accessible.